prisma-fmt/src/lib.rs

```diff
@@ -65,6 +66,28 @@ pub fn lint(schema: String) -> String {
     lint::run(&schema)
 }
 
+/// Function that emulates what happened when improperly calling the `getDmmf` query engine RPC on invalid schemas.
+/// When the schema is valid, nothing happens.
+/// When the schema is invalid, the function displays a human-friendly error message indicating the schema lines
+/// where the errors lie and the total error count, e.g.:
+///
+/// ```sh
+/// The `referentialIntegrity` and `relationMode` attributes cannot be used together. Please use only `relationMode` instead.
+///   -->  schema.prisma:5
+///   |
+/// 4 |   relationMode         = "prisma"
+/// 5 |   referentialIntegrity = "foreignKeys"
+/// 6 | }
+///   |
+///
+/// Validation Error Count: 1
```

Comment:
The validation errors count is a quirk of the query engine. Does it make sense, from a UX perspective? It's not really for us to answer but since we're implementing a new function we should ask the question.

Reply:
It's something the users have come to expect from the output, and something we're keeping track of in e.g. `introspection-ci`. I think they're useful and should stay in the output